conformance-suites/2.0.0/conformance2/textures/misc/copy-texture-image.html

```diff
@@ -80,8 +80,12 @@
         "RG8",
         "RGB8",
         "RGBA8",
+        "R8_SNORM",
```

Comment:
Looks like it was purposeful and explicit: https://cvs.khronos.org/bugzilla/show_bug.cgi?id=10001